### DISCUSSION TITLE
Applying different  Animations to a model

### DISCUSSION THREAD

**Discussion by ozair-qayam** [google/filament#6434] [General]
I have two .glb file,then parse them to two Renderables(A,B),

the Renderables A use to ModelViewer.
the Renderables B parse one animations.
what should i do to use B's aimations in A ModelViewer?

I try to use B‘s animator applyAnimation() , the animtion stop and cant play B's animation

**Comment by dandingol03** [comment]
why there is no particiation?

**Reply by ninjz** [reply]
@ozair-qayam Were you able to figure this out?

**Reply by jellychen** [reply]
@ozair-qayam  Were you able to figure this out?
